sdk/include/psdk/shlwapi.h

```diff
@@ -1892,6 +1892,13 @@ SHLockShared(
     _In_  DWORD dwProcessId
     );
 
+PVOID
+WINAPI
+SHLockSharedEx(
```

Comment:
This doesn't belong here. undocshell.h perhaps?

Reply:
Moved.